WordPress/gutenberg
# Make the Site Logo block placeholder state smaller

## What?
Closes #48217 by making the Site Logo block's placeholder state smaller — from 120px by 120px to 60px by 60px.

## Why?
The original intent for `120px` was to match the default image width that is applied when an image is used as the logo. But as a 1:1 ratio, 120px by 120px does not represent a common logo height, and it distorts the vertical spacing of header patterns (causing substantial vertical shift when an image is applied).

Maintaining a 1:1 is important as it helps to differentiate the site logo block from other image placeholders. 

The current method for getting around this is setting the width within the block's attributes, although then the applied image will only render at 60px wide. 

## How?
SCSS change. 

## Testing Instructions

1. Add a Site Logo block, without a site logo uploaded.
2. See a smaller placeholder.

## Screenshots or screencast 

Current: 

<img width="1507" alt="CleanShot 2023-02-17 at 16 23 25" src="https://user-images.githubusercontent.com/1813435/219796227-7bcc313f-e8cd-45b1-9d5b-d5972e6df21b.png">

Proposed: 

<img width="1506" alt="CleanShot 2023-02-17 at 16 22 53" src="https://user-images.githubusercontent.com/1813435/219796146-1b084311-448e-47aa-a83b-f2e734113319.png">
